packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -1027,6 +1028,41 @@ export class Mulw extends Op {
   }
 }
 
+// A,B / C. Fail if C == 0 or if result overflows.
+// The notation A,B indicates that A and B are interpreted as a uint128 value,
+// with A as the high uint64 and B the low.
+// push to stack [...stack, bigint]
+// Availability: v6
+export class Divw extends Op {
+  readonly line: number;
+  /**
+   * Asserts 0 arguments are passed.
+   * @param args Expected arguments: [] // none
+   * @param line line number in TEAL file
+   */
+  constructor (args: string[], line: number) {
+    super();
+    this.line = line;
+    assertLen(args.length, 0, line);
+  };
+
+  execute (stack: TEALStack): void {
```

Comment:
Can you explain for me what `execute` is?(Is it a class, a method, a function???)

Reply:
This is method in class. You can read more details at [here](https://www.typescriptlang.org/docs/handbook/classes.html)